benchmark/synthetic_benchmark.py

```diff
@@ -0,0 +1,180 @@
+import argparse
+import torch.backends.cudnn as cudnn
+import torch.nn.functional as F
+import torch.optim as optim
+import torch.utils.data.distributed
+from torchvision import models
+
+# import horovod.torch as hvd
+import timeit
+import numpy as np
+
+# 导入bagua包
```

Comment:
do not mix languages

Reply:
sorry I thought the PR is ready. nevermind. request review again when ready

Reply:
Yes I found this problem and it is being fixed